packages/ui/src/styles/globals.css

```
@@ -126,6 +127,88 @@
     }
 }
 
+/* Import highlight.js GitHub theme - only loads in light mode */
+@import 'highlight.js/styles/github.css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)